ruby/book.rb

```diff
@@ -0,0 +1,9 @@
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+end
```

Comment:
- [ ] It seems that you forgot to create the custom method `add_rental` in the Book class to make a person be able to rent a book. Please do so and make sure to test it out in your terminal. 🙏

Reply:
Thanks very much for the review @MahmoudBakr23, I will implement that now.